packages/fxa-payments-server/src/store/selectors.ts

```diff
@@ -1,29 +1,29 @@
-import { Selector, Plan } from './types';
+import { Plan, State } from './types';
 
-export const profile: Selector = state => state.api.profile;
-export const token: Selector = state => state.api.token;
-export const subscriptions: Selector = state => state.api.subscriptions;
-export const plans: Selector = state => state.api.plans;
-export const customer: Selector = state => state.api.customer;
+export const profile = (state: State) => state.api.profile;
```

Comment:
At one point, I thought this generic `Selector` type was a good idea. But, letting the specific function signatures of these selectors come through turns out to be handy for type inference everywhere else.